bears/python/PyCheckerBear.py

```diff
@@ -0,0 +1,41 @@
+import shlex
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+        DistributionRequirement)
+
+
+@linter(executable='pychecker',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.py):(?P<line>\d+): '
+                     r'(?P<message>.*)')
+class PyCheckerBear:
+    """
+    Find bugs in your python source code.
+
+    The code for each function, class, and method is checked for possible
+    problems.Checks for unused globals and locals(module or variable),unused
+    method arguments and using a variable before setting it or if you are
```

Comment:
a single space after a comma and a full-stop looks way nicer ;)

Reply:
:+1: